x-pack/plugins/actions/server/types.ts

```diff
@@ -136,3 +136,8 @@ export interface ProxySettings {
   proxyHeaders?: Record<string, string>;
   proxyRejectUnauthorizedCertificates: boolean;
 }
+
+export interface ResponseSettings {
+  maxResponseContentLength: number;
+  responseTimeout: number;
```

Comment:
**nit:** Would it be cleaner like this? so it can be used like `getResponseSettings().timeout`, etc?

```suggestion
  maxContentLength: number;
  timeout: number;
```